@xen-orchestra/backups/writers/DeltaBackupWriter.js

```diff
@@ -69,6 +71,36 @@ exports.DeltaBackupWriter = class DeltaBackupWriter extends MixinBackupWriter(Ab
     return this._cleanVm({ merge: true })
   }
 
+  healthCheck(sr) {
```

Comment:
Adds it to https://github.com/vatesfr/xen-orchestra/blob/master/%40xen-orchestra/backups/docs/VM%20backups/README.md#writer-api :slightly_smiling_face: